### DISCUSSION TITLE
brain.recurrent.LSTM() is not working

### DISCUSSION THREAD

**Discussion by anastasiiamay** [BrainJS/brain.js#732] [Q&A] [answered]
I took an example from brain.js.org, but got error when tried to use it. Error occurs because of this line:
`const lstm = new brain.recurrent.LSTM();`
Error is:
`Cannot read property 'LSTM' of undefined`
So I tried to run your example on your website. And also got error:
`Uncaught TypeError: Cannot read properties of undefined (reading 'LSTM')`
Can someone tell me please am I doing something wrong?

**Comment by robertleeplummerjr** [comment, marked as answer]
Try now.  There were a lot of complications when we upgraded to Typescript.  Also, if you continue to have issues, post some of your training data.

**Reply by SudhanshuBlaze** [reply]
I am using it react.js, again getting the same error

```
"react": "^18.2.0",
"react-dom": "^18.2.0",
```

**Comment by ahmed-abdelfata7** [comment]
I have the same issue also 
```js
import brain from "brainjs";

const trainingData = [
  [1, 2, 3, 4, 5],
  [5, 4, 3, 2, 1],
];
const net = new brain.recurrent.LSTM();
net.train(trainingData, {
  log: (status) => {
    console.log(status);
  },
});
```
`TypeError: Cannot read properties of undefined (reading 'recurrent')`

**Reply by Radulepy** [reply]
> I have the same issue also
> 
> ```js
> import brain from "brainjs";
> 
> const trainingData = [
>   [1, 2, 3, 4, 5],
>   [5, 4, 3, 2, 1],
> ];
> const net = new brain.recurrent.LSTM();
> net.train(trainingData, {
>   log: (status) => {
>     console.log(status);
>   },
> });
> ```
> 
> `TypeError: Cannot read properties of undefined (reading 'recurrent')`

Hi, I've fixed it by using default Module selector:
`const net = new brain.default.recurrent.LSTM();`
